src/test/scenarios/datafactory/output/src/datafactory-preview/azext_datafactory_preview/tests/latest/test_datafactory_scenario.py

```diff
@@ -112,13 +148,53 @@ def call_scenario(test, rg):
     cleanup_scenario(test, rg)
 
 
+@try_manual
+def call_scenario_min(test, rg):
+    setup_scenario(test, rg)
+    step_create_min(test, rg, checks=[])
+    step_show_min(test, rg, checks=[])
+    step_list_min(test, rg, checks=[])
+    step_list2_min(test, rg, checks=[])
+    step_update_min(test, rg, checks=[])
+    step_configure_factory_repo_min(test, rg, checks=[])
+    step_get_data_plane_access_min(test, rg, checks=[])
+    step_get_git_hub_access_token_min(test, rg, checks=[])
+    step_integration_runtime_self_hosted_create_min(test, rg, checks=[])
+    step_integration_runtime_show_min(test, rg, checks=[])
+    step_integration_runtime_list_min(test, rg, checks=[])
+    step_integration_runtime_update_min(test, rg, checks=[])
+    step_integration_runtime_linked_min(test, rg, checks=[])
+    step_integration_runtime_get_connection_info_min(test, rg, checks=[])
+    step_integration_runtime_get_monitoring_data_min(test, rg, checks=[])
+    step_integration_runtime_get_status_min(test, rg, checks=[])
+    step_integration_runtime_list_auth_key_min(test, rg, checks=[])
+    step_integration_runtime_regenerate_auth_key_min(test, rg, checks=[])
+    step_integration_runtime_remove_link_min(test, rg, checks=[])
+    step_integration_runtime_start_min(test, rg, checks=[])
+    step_integration_runtime_stop_min(test, rg, checks=[])
+    step_integration_runtime_sync_credentials_min(test, rg, checks=[])
+    step_integration_runtime_upgrade_min(test, rg, checks=[])
+    step_integration_runtime_delete_min(test, rg, checks=[])
+    step_trigger_create_min(test, rg, checks=[])
+    step_trigger_update_min(test, rg, checks=[])
+    step_trigger_show_min(test, rg, checks=[])
+    step_trigger_list_min(test, rg, checks=[])
+    step_trigger_get_event_subscription_status_min(test, rg, checks=[])
+    step_trigger_query_by_factory_min(test, rg, checks=[])
+    step_trigger_start_min(test, rg, checks=[])
+    step_trigger_stop_min(test, rg, checks=[])
+    step_trigger_subscribe_to_event_min(test, rg, checks=[])
+    step_trigger_unsubscribe_from_event_min(test, rg, checks=[])
+    step_trigger_delete_min(test, rg, checks=[])
+    step_delete_min(test, rg, checks=[])
+    cleanup_scenario(test, rg)
```

Comment:
If user has customization steps about their scenario. Will minimal scenario also be impact ?

Reply:
I'm also hesitated on this.    A pro of doing is that it make overriding on normal and minimal testwith different parameters  more easily.